android/app/src/test/java/com/microsoft/codepush/common/datacontracts/DataContractsUnitTests.java

```diff
@@ -70,8 +68,9 @@ public void dataContractsTest() throws Exception {
         codePushDeploymentStatusReport.setPreviousDeploymentKey(PREVIOUS_DEPLOYMENT_KEY);
         codePushDeploymentStatusReport.setPreviousLabelOrAppVersion(PREVIOUS_LABEL);
         codePushDeploymentStatusReport.setStatus(CodePushDeploymentStatus.SUCCEEDED);
-        codePushDeploymentStatusReport.setLocalPackage(codePushLocalPackage);
-        assertEquals(codePushLocalPackage, codePushDeploymentStatusReport.getLocalPackage());
+        // TODO there should be CodePushPackage instance instead!!!
```

Comment:
Could you please clarify why?

Reply:
@sergey-akhalkov because that was the logical mistake I previously fixed in the main code, but but not in tests.